tests/test_migrators.py

```diff
@@ -1613,8 +1613,97 @@
     - mpmath
 """
 
+version_license="""
+{% set version = "0.8" %}
+
+package:
+  name: viscm
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/v/viscm/viscm-{{ version }}.tar.gz
+  sha256: 5a9677fa4751c6dd18a5a74e7ec06848e4973d0ac0af3e4d795753b15a30c759
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+    - numpy
+  run:
+    - python
+    - numpy
+    - matplotlib
+    - colorspacious
+
+test:
+  imports:
+    - viscm
+
+about:
+  home: https://github.com/bids/viscm
+  license: MIT
+  license_family: MIT
+  # license_file: '' we need to an issue upstream to get a license in the source dist.
+  summary: A colormap tool
+
+extra:
+  recipe-maintainers:
+    - kthyng
+"""
+
+version_license_correct="""
+{% set version = "0.9" %}
+
+package:
+  name: viscm
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/v/viscm/viscm-{{ version }}.tar.gz
+  sha256: c770e4b76f726e653d2b7c2c73f71941a88de6eb47ccf8fb8e984b55562d05a2
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+    - numpy
+  run:
+    - python
+    - numpy
+    - matplotlib
+    - colorspacious
+
+test:
+  imports:
+    - viscm
+
+about:
+  license_file: LICENSE
```

Comment:
Can this be added after `license` entry?

Reply:
It's a bit more annoying to code up, but if you can put in the needed regex replacement [here](https://github.com/regro/cf-scripts/pull/637/files#diff-65ad1c9390fe432dca7ca211091e3287R122-R124) that would be spectacular.